app/models/authenticator/database.rb

```diff
@@ -13,7 +13,12 @@ def uses_stored_password?
     def _authenticate(username, password, _request)
       user = case_insensitive_find_by_userid(username)
 
-      user && user.authenticate_bcrypt(password)
+      return [nil, _('Your account has been locked due to too many failed login attempts, please contact the administrator.')] if user&.locked?
```

Comment:
Should this return [false, "..."] to be consistent with the one below that also returns false?